cpp2python.py

```diff
@@ -278,6 +295,92 @@ def process_line(line):
                 '\s*([^,]+)\s*,\s*' + \
                 'S[A-Z]+\s*\(\s*([\w\d]+)[^\)]+\)\s*\)\s*\)',
               '\\1.\\2.connect(\\3.\\4)', line)
+    
+
+    """ alter for statement
+    for(j = 1; j < c - 1; j+=1)
+                v
+    for j in range(1,c-1,1):
+        
+    """
+    regex = r"for\s?\(\s?(\s?(\w+)\b\s?=\s?(\d+|\w+)\b\s?)?;\s?((\w+)\s?(>|>=|<|<=|==|!=)?\s?((\w+|\d+)?\s?(\-|\+|\*|\/)?\s?(\w+|\d+)?))?\s?;\s?(\w+\s?(\+\+|\-\-|\+|\-)(\d+)?(\w+)?)?\s?\)"
+    subst = "for \\2 in range(\\3, \\7):"
+    line = re.sub(regex, subst, line)
+
+    """ remove type
+
+        int|float|double|char i;
+                V
+        i
+    """
+    line = re.sub('(int|double|float|char) ', '', line) # 
```

Comment:
Couldn't we join this reg exp with reg exp which removes pointers and  write something generic like 
[optional const] [optional unsigned] [standard type name] [optional star, at least one space before or after star, or just space without star] [variable name which starts from letter or underscore and might contain digits] ?